scripts/create-protection-market-curve3Crypto.ts

```diff
@@ -0,0 +1,127 @@
+const chainId = getChainId(hre);
+const { AddressZero } = hre.ethers.constants;
+
+// STEP 1: TRIGGER CONTRACT SETUP
+const name = 'Curve 3Crypto Trigger'; // name
+const symbol = 'CURVE-3CRYPTO-TRIG'; // symbol
+const description = "Triggers when the Curve 3Crypto pool's virtual price decreases by more than 50% between consecutive checks, or the internal balances tracked in the Curve 3Crypto pool are more than 50% lower than the true balances"; // prettier-ignore
+const platformIds = [3]; // platform ID for Curve
+const recipient = '0xSetRecipientAddressHere'; // subsidy recipient
+
+// STEP 2: TRIGGER CONTRACT DEVELOPMENT
+// For this step, see the ITrigger.sol and MockTrigger.sol examples and the corresponding documentation
+
+// STEP 3: PROTECTION MARKET DEPLOYMENT
+async function main(): Promise<void> {
+  const curvePoolAddress = String(process.env.CURVE_POOL);
+  if (!curvePoolAddress) throw new Error("Please define the 'CURVE_POOL' environment variable.");
+
+  if (!utils.isAddress(recipient)) throw new Error('\n\n**** Please set the recipient address on line 23 ****\n');
+
+  // Compile contracts to make sure we're using the latest version of the trigger contracts
+  await hre.run('compile');
+
+  const overrides = { gasPrice: await getGasPrice() };
+
+  // VERIFICATION
+  // Verify the user is ok with the provided inputs
+  console.log(chalk.bold.yellow('\nPLEASE VERIFY THE BELOW PARAMETERS\n'));
+  console.log('  Deploying protection market for:   Curve 3Crypto');
+  console.log(`  Deployer address:                  ${signer.address}`);
+  console.log(`  Deploying to network:              ${hre.network.name}`);
+  console.log(`  Gas Price:                         ${JSON.stringify(overrides.gasPrice)}`);
+
+  const response = await waitForInput('\nDo you want to continue with deployment? y/N\n');
+  if (response !== 'y') {
+    logFailure('\nUser chose to cancel deployment. Exiting script');
+    return;
+  }
+  logSuccess('Continuing with deployment...\n');
+
+  // DEPLOY INTEREST RATE MODEL
+  // Get instance of the Trigger ContractFactory with our signer attached
+  const irModelFactory: ContractFactory = await hre.ethers.getContractFactory('JumpRateModelV2', signer);
+
+  // Deploy the interest rate model, configured with the following parameters:
+  //   - 2% base borrow rate at zero utilization
+  //   - 2% borrow rate at 80% utilization (no increase)
+  //   - Linear increase from 2% to 15% borrow rate at 100% utilization
+  const constructorArgs = [
+    '20000000000000000', // baseRatePerYear of 2% = 2e16
+    '0', // multiplierPerYear of 0% = 0 gives 2% borrow rate at kink
+    '650000000000000000', // jumpMultiplierPerYear of 65% = 6.5e17 gives 15% borrow rate at 100% utilization
+    '800000000000000000', // kink of 0.8 = 8e17 = sets the model kink at 80% utilization
+    '0x1725d89c5cf12F1E9423Dc21FdadC81C491a868b', // Cozy multisig
```

Comment:
This address is the cozy multisig on mainnet, on arbitrum it's this:
```suggestion
    '0xe570e347932621d08c242035c56d79a33f4269af', // Cozy multisig
```

Reply:
I opted for making it an arg, since the script can be used on mainnet or arbitrum. Also added sample commands which show usage of both